File: WebAppClient/src/app/contracts/order/order_detail.ts
export class Order_Detail {
    id: string;
    orderCode: string;
    address: string;
    description: string;
    dateCreated: Date;
    orderBasketItems: OrderBasketItem[];
}

export class OrderBasketItem_VM {
    imagePath: string;
    productName: string;
    productDescription: string;
    itemPrice: string;
    quantity: number;
    rating: Partial<any>;
    totalItemAmount: string;
}

export class OrderBasketItem {
    name: string;
    description: string;
    price: number;
    quantity: number;
    rating: number;
    orderProductImageFile: OrderProductImageFile;
}

export class OrderProductImageFile {
    productImageFileId: string;
    fileName: string;
    path: string;
}

File: WebAppAPI/Core/WebAppAPI.Application/DTOs/Order/OrderDetail.cs
using WebAppAPI.Domain.Entities;

namespace WebAppAPI.Application.DTOs.Order
{
    public class OrderDetail
    {
        public string Id { get; set; }
        public string OrderCode { get; set; }
        public string Address { get; set; }
        public string Description { get; set; }
        public DateTime DateCreated { get; set; }
        public int StatusId { get; set; }
        public List<OrderItems> OrderBasketItems { get; set; }
    }
}
